weave/weaveflow/__init__.py

```diff
@@ -16,5 +16,3 @@
 from .openai import *
 from .anyscale import *
 from .huggingface import *
```

Comment:
any reason for keeping huggingface and anyscale here still? I feel like we want to keep weaveflow as lean as possible

Reply:
This weaveflow directory is going to away once I fully switch stuff to pydantic, so that stuff will dissappear. Though I guess we don't actually import huggingface stuff since we're not getting a crash there.